types.d.ts

```diff
@@ -0,0 +1,53 @@
+export abstract class Serializable {
```

Comment:
This needs to be renamed to `index.d.ts` and wrapped in `declare module "ts_serialize" {` so that typescript will automatically load the types for the `dist` module, and when a user runs `import { Serializable, SerializeProperty } from "ts_serialize";` it will be associated with these definitions